### PR TITLE
Add sticker like images

### PR DESCRIPTION
**Pull request by hroncok**
More images will follow

### CODE REVIEW

**Comment by hroncok** [comment]
Not all of them are perfect, but they can always be remade in the future.

File: points.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@
   ready: false
   content: |
     All Pythons, all the way down. Well, until you get to C.
+  logo: fedoralovespython
```
Comment (encukou) [inline points.yml]: Maybe Koji+Bodhi+COPR logos would be appropriate here?

Comment (hroncok) [reply points.yml]: Will try that.

Comment (hroncok) [reply points.yml]: Pushed

**Comment by encukou** [comment]
Looks good.

I was just thinking about using the `F♥P` for the last point, instead of `...`. It's relevant, and closing with a reference to the beginning is always good in a story :)

**Comment by hroncok** [comment]
Done

**Pull request merged**